### PR TITLE
Add NCD logo to Hardware partners section

### PR DESCRIPTION
**Pull request by Yndira-E**
## Description

Added the NCD.io logo to `/partners` in the Hardware section

## Related Issue(s)

https://github.com/FlowFuse/customer/issues/183

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/partners/index.njk [hunk 1]
```diff
@@ -27,10 +27,13 @@ description: FlowFuse partners with leading cloud, hardware, technology and cons
             <div class="flex flex-col md:flex-row md:gap-10 gap-6">
             <p class="md:w-[65%] w-full">FlowFuse partners with key industry hardware partners to provide an integrated experience
                 using Node-RED and FlowFuse.</p>
-            <ul class="md:w-[35%] flex flex-row justify-center md:justify-end w-full items-center">
+            <ul class="md:w-[35%] flex flex-row justify-center md:justify-end w-full items-center gap-10">
                 <li class="w-[150px] pb-6 md:pb-0 hover:transition-opacity hover:opacity-75 transition ease-in duration-300">
                     <a href="/partners/ctrlx/">{% image "../images/partners/logo-rexroth.png", "Rexroth logo", [150] %}</a>
                 </li>
+                <li class="w-[150px] pb-6 md:pb-0 hover:transition-opacity hover:opacity-75 transition ease-in duration-300">
+                    <a href="/partners/ctrlx/">{% image "../images/partners/logo-ncd-io.png", "Rexroth NCD.io", [150] %}</a>
```
Comment (ZJvandeWeg) [inline src/partners/index.njk:35]:
```suggestion
                    <a href="/partners/ctrlx/">{% image "../images/partners/logo-ncd-io.png", "NCD.io logo", [150] %}</a>
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/partners/index.njk [hunk 2]
```diff
@@ -32,7 +32,7 @@ description: FlowFuse partners with leading cloud, hardware, technology and cons
                     <a href="/partners/ctrlx/">{% image "../images/partners/logo-rexroth.png", "Rexroth logo", [150] %}</a>
                 </li>
                 <li class="w-[150px] pb-6 md:pb-0 hover:transition-opacity hover:opacity-75 transition ease-in duration-300">
-                    <a href="/partners/ctrlx/">{% image "../images/partners/logo-ncd-io.png", "Rexroth NCD.io", [150] %}</a>
+                    <a href="/partners/ctrlx/">{% image "../images/partners/logo-ncd-io.png", "NCD.io logo", [150] %}</a>
```
Comment (ZJvandeWeg) [inline src/partners/index.njk:35]: @Yndira-E Please look at the href too:

```suggestion
                    <a href="https://ncd.io/">{% image "../images/partners/logo-ncd-io.png", "NCD.io logo", [150] %}</a>
```

Comment (Yndira-E) [reply src/partners/index.njk:35]: Thanks @ZJvandeWeg, and sorry! Seems like I'm trying to go too fast

File: src/partners/index.njk [hunk 3]
```diff
@@ -27,10 +27,13 @@ description: FlowFuse partners with leading cloud, hardware, technology and cons
             <div class="flex flex-col md:flex-row md:gap-10 gap-6">
             <p class="md:w-[65%] w-full">FlowFuse partners with key industry hardware partners to provide an integrated experience
                 using Node-RED and FlowFuse.</p>
-            <ul class="md:w-[35%] flex flex-row justify-center md:justify-end w-full items-center">
+            <ul class="md:w-[35%] flex flex-row justify-center md:justify-end w-full items-center gap-10">
                 <li class="w-[150px] pb-6 md:pb-0 hover:transition-opacity hover:opacity-75 transition ease-in duration-300">
                     <a href="/partners/ctrlx/">{% image "../images/partners/logo-rexroth.png", "Rexroth logo", [150] %}</a>
                 </li>
+                <li class="w-[150px] pb-6 md:pb-0 hover:transition-opacity hover:opacity-75 transition ease-in duration-300">
+                    <a href="https://ncd.io/">{% image "../images/partners/logo-ncd-io.png", "NCD.io logo", [150] %}</a>
```
Comment (Yndira-E) [inline src/partners/index.njk:35]:
```suggestion
                    <a href="https://ncd.io/" target="_blank">{% image "../images/partners/logo-ncd-io.png", "NCD.io logo", [150] %}</a>
```

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**